ui/pom.xml

```diff
@@ -0,0 +1,275 @@
+<?xml version="1.0"?>
+<project
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+        xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
```

Comment:
This one doesn't exist at the given location. Best probably to remove any parent relationship for now and make this one self-contained.

Reply:
Yeah, the pom I just copied from the other repo and hadn't looked at it too closely.

Reply:
Ok, are you planning to do this as part of this PR? Or rather would like to have this merged first and follow up on this one separately? Or want me to take a look? Let me know :)

Reply:
I was thinking to get this in and then follow up with changes.  Haven't looked at the maven build much yet, just looked at getting a dev ui up for now.  I can do some more cleanup on this today, then maybe just merge this tomorrow and work from there?  We can always re-organize if needed.

Reply:
Ok, cool. Then I'd say do the changes and clean-up you have in mind and let me know so I can merge it. I'll take a look into the service registry's server side part of the built in the mean time.

Reply:
@gunnarmorling I pushed a few updates per your and @indraraj comments.  I'm not real clear on build / deployment with quarkus, but we can figure out and rearrange if needed as we progress.